### PR TITLE
DHFPROD-8269: Use proper endpoints, instead of mock data, for the entity specific data in the side-bar in explore screen

### PR DESCRIPTION
**Pull request by aniamy**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- N/A Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- N/A Added Release Notes

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/pages/Browse.test.tsx [hunk 1]
```diff
@@ -5,12 +5,36 @@ import {MemoryRouter} from "react-router-dom";
...
+import axiosMock from "axios";
+import {act} from "react-dom/test-utils";
 
 jest.mock("axios");
 jest.setTimeout(30000);
 
-
 describe("Explorer Browse page tests ", () => {
+
+  const modelsResponse = [
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/pages/Browse.test.tsx:16]: Looks like we are already storing mock entity model responses and sample hub central config responses in few files under `src/assets/mock-data/modeling/modeling.js`  and `src/assets/mock-data/explore/model-response.js`.
Can we place this mock data as well at one of those appropriate places and refer them here, as needed. ?

File: marklogic-data-hub-central/ui/src/pages/Browse.test.tsx [hunk 2]
```diff
@@ -5,12 +5,36 @@ import {MemoryRouter} from "react-router-dom";
...
+              "datatype": "string",
+              "collation": "http://marklogic.com/collation/codepoint"
+            }
+          }
+        }
+      }
+
+    }
+  ];
+  const hubConfigResponse = {"modeling": {"entities": {"Customer": {"graphX": 16, "graphY": 54, "color": "#e3deeb", "icon": "FaOdnoklassniki"}}}};
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/pages/Browse.test.tsx:37]: This also can be moved under `src/assets`, like mentioned in the previous comment.

**Review by xnikhil08 — CHANGES_REQUESTED** [review]
Looks good. Just mentioned few suggestions about moving the mock data under src/assets, to follow the UI standards.

**Comment by briantang** [comment]
Thanks @xnikhil08, I was giving out a hand in that department and somehow left behind some remnants there. Refactored into a common area now.

**Review by xnikhil08 — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@aniamy two UI tests are failing on this PR https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-6455/11/tests, Please look into it

**Review by Sanjeevani19 — APPROVED** [review]
Looks good, tests passed on latest build

**Comment by briantang** [comment]
Pushed an e2e fix for the failing entitiesSideBar.spec test. Should pass now.

**Review by briantang — APPROVED** [review]

**Review by xnikhil08 — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**